caas/kubernetes/provider/customresourcedefinitions.go

```diff
@@ -26,14 +26,14 @@ import (
 
 //go:generate mockgen -package mocks -destination mocks/crd_getter_mock.go github.com/juju/juju/caas/kubernetes/provider CRDGetterInterface
 
-func (k *kubernetesClient) getCRDLabels(appName string) map[string]string {
+func (k *kubernetesClient) getCRDLabelsGlobal(appName string) map[string]string {
```

Comment:
We have been put lots of effects to make the label clean enough rather than put anything possible for any resources.
For example, for namespaces resources, they are already in a specific namespace. 
So there is no need to put model-uuid/model-name stuff into the labels.
This is the general rule for labeling on any already supported resources by Juju.